graal-js/src/com.oracle.js.parser/src/com/oracle/js/parser/Parser.java

```diff
@@ -3625,10 +3628,23 @@ private Expression primaryExpression(boolean yield, boolean await) {
                 markThis();
                 return new IdentNode(primaryToken, finish, name).setIsThis();
             case IDENT:
+                // mimic code from AbstractParser.getIdent()
+                final long identToken = token;
+                String identString = (String) getValue(identToken);
+
+                if (identString.equals("module") && lookahead().equals(LBRACE)) {
+                    nextOrEOL();
+                    expect(LBRACE);
+
+                    return module(source.getName() + "#L" + line + "T" + start, true);
```

Comment:
this `getName() + "#L" + line + ...` looks like some debugging code? I think it should be removed

Reply:
This was inserted due to an issue in the doc repository for testing reasons. It might have to come back some day if the developers decide to use something similar, for right now I deleted it.